doc/source/back-end/tutorialv2.rst

```diff
@@ -54,116 +156,153 @@ ces deux règles sont respectées).
-    introductions et conclusions éventuelles. L'implémentation de ZdS considère 
-    que ceux-ci sont ``introduction`` et ``conclusion``, mais ce n'est pas 
-    obligatoire.
+.. attention::
 
-En fonction de sa position dans l'arborescence du contenu, un conteneur peut 
-aussi bien représenter le tutoriel/article lui-même (s'il est conteneur 
-principal), une partie ou un chapitre. Ainsi, dans l'exemple suivant :
+    L'introduction et la conclusion d'un conteneur possèdent également un 
+    *slug*, pour des raisons de stockage (voir plus bas). Il ne faut pas 
+    oublier la contrainte d'unicité à l'intérieur d'un conteneur.
 
-.. sourcecode:: text
+Cycle de vie des contenus
+==========================
 
-    Contenu (tutoriel)
-    |
-    +-- Conteneur 1
-         |
-         +-- Extrait 1
+Quelque soit le type de contenu, le cycle de vie de celui-ci reste toujours le même.
+Un contenu peut être rédigé par un ou plusieurs auteurs. Chaque modification 
+est conservée afin de pouvoir retrouver l'historique des modifications et éventuellement
+récupérer un morceau perdu. Lorsqu'un contenu est créé il rentre dans sa première étape.
 
+Le brouillon
+------------
 
-le ``Conteneur 1`` sera rendu par ZdS comme étant un chapitre d'un (moyen-) 
-tutoriel, alors que dans l'exemple suivant :
+Le brouillon est la première étape du cycle de vie d'un contenu. Il donne 
+toujours l'état le plus récent d'un contenu vu par les auteurs. Chaque fois 
+que le contenu est modifié, c'est la version brouillon qui est mise à jour. 
+La version brouillon est accessible uniquement pour les auteurs et validateurs 
+d'un tutoriel. Si on souhaite donner un accès en lecture seule à nos écrits, 
+il faut passer par la méthode adéquate.
 
-.. sourcecode:: text
+La bêta
+-------
 
-    Contenu (tutoriel)
-    |
-    +-- Conteneur 1
-         |
-         +-- Conteneur 2
-         |     |
-         |     +-- Extrait 1
-         |
-         +-- Conteneur 3
-               |
-               +-- Extrait 1
+Lorsque les auteurs estiment que leur tutoriel a atteint un certain niveau de 
+maturité, et qu'ils souhaitent recueillir des retours de la communauté, ils ont 
+la possibilité de le mettre à la disposition de cette dernière le contenu en
+lecture seule. C'est le mode bêta.
 
+Lors de la mise en bêta d'un contenu, un sujet est automatiquement ouvert dans 
+la Bêta-zone.
 
-le ``Conteneur 1`` sera rendu par ZdS comme étant une partie d'un (big-) 
-tutoriel, et ``Conteneur 2`` et ``Conteneur 3`` comme étant les chapitres de 
-cette partie.
+Il faut en outre noter que seule une version précise du contenu est mise en 
+bêta. Au moment de la mise en bêta, les versions brouillon et bêta coïncident 
+mais l'auteur peut tout à fait poursuivre son travail sans affecter la seconde. 
+Seulement, la version brouillon ne sera plus identique à la version en bêta et 
+il ne faudra pas oublier de mettre à jour cette dernière pour que la communauté 
+puisse juger des dernières modifications.
```

Comment:
Sans la 301, je n'aurais pas compris l'intérêt du slug. Un nom de dossier peut bien être un entier, non ?

----- Mail original -----
De: "Pierre Beaujean" <notifications@github.com>
À: "artragis/zds-site" <zds-site@noreply.github.com>
Cc: "Vincent" <vincent.lefoulon@free.fr>
Envoyé: Mardi 19 Mai 2015 22:05:39
Objet: Re: [zds-site] Doc (#130)



In doc/source/back-end/tutorialv2.rst : >  
> -le ``Conteneur 1`` sera rendu par ZdS comme étant une partie d'un (big-) 
> -tutoriel, et ``Conteneur 2`` et ``Conteneur 3`` comme étant les chapitres de 
> -cette partie.
> +Il faut en outre noter que seule une version précise du contenu est mise en 
> +bêta. Au moment de la mise en bêta, les versions brouillon et bêta coïncident 
> +mais l'auteur peut tout à fait poursuivre son travail sans affecter la seconde. 
> +Seulement, la version brouillon ne sera plus identique à la version en bêta et 
> +il ne faudra pas oublier de mettre à jour cette dernière pour que la communauté 
> +puisse juger des dernières modifications. 

Le pk est super utile pour les bases de données (c'plus court de chercher un entier qu'une chaine de caractère). À l'inverse, le slug nous sert pour la partie "fichier': comme dit plus haut, c'est le nom du dossier dans lequel sera rangé tout le bordel. En plus de ça, il sert à détecter qu'un visiteur demande pas des trucs fantaisistes ou quand balancer une HTTP 301 :) 

— 
Reply to this email directly or view it on GitHub .